### PR TITLE
add support for visit_id and dltoken

### PR DESCRIPTION
**Pull request by oremj**
Add support for visit_id parameter. Supports: https://bugzilla.mozilla.org/show_bug.cgi?id=1677497

### CODE REVIEW

File: testing/stubtestclient/main.go [hunk 1]
```diff
@@ -56,6 +57,7 @@ func init() {
 	flag.StringVar(&experiment, "experiment", "exp1", "experiment")
 	flag.StringVar(&variation, "variation", "var1", "variation")
 	flag.StringVar(&installerType, "installer_type", "full", "installer_type")
+	flag.StringVar(&installerType, "visit_id", "vid", "visit_id")
```
Comment (pmac) [inline testing/stubtestclient/main.go:60]: Should it be `&visitId` as the first argument?

**Review by pmac — APPROVED** [review]
Looks good. Just one question.

File: testing/stubtestclient/main.go (hunk 1 above)
Comment (oremj) [reply testing/stubtestclient/main.go:60]: Thanks, that was a typo!

**Comment by oremj** [comment]
@pmac I've added the remaining features to this PR. Will you do one more review please?

**Review by pmac — APPROVED** [review]
I'm admittedly not the best at Go, but this looks good to me. It looks like you've refactored some things to make them more flexible for future while doing this, which seems good. I see that the site should send you a `visit_id` with the other stub attribution data. I don't see any obvious problems. Nice work 🍰

**Pull request merged**